hypershift-operator/featuregate/feature.go

```diff
@@ -0,0 +1,27 @@
+package featuregate
+
+import (
+	"k8s.io/apimachinery/pkg/util/runtime"
+	"k8s.io/component-base/featuregate"
+)
+
+const (
+	// OpenStack is a feature gate for running clusters on OpenStack.
+	// owner: @username
```

Comment:
Could Emilien and/or Matthew be added here? Or would we want to add a team alias here?

Reply:
let me follow up to introduce team alias.